_data/Mehulipal/Mehulipal.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mehuli Pal
+institution: Indian Institute of Technology, Patna
+profile_pic: mehulipal.jpg
```

Comment:
image file path name seems incorrect to me, please change it to `Mehulipal.jpg`

Reply:
Done the requested changes...